ldes-server-admin/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/domain/eventstream/entities/EventStreamTest.java

```diff
@@ -44,11 +42,11 @@ static class EventStreamArgumentProvider implements ArgumentsProvider {
 		public Stream<Arguments> provideArguments(ExtensionContext extensionContext) throws Exception {
 			return Stream.of(
 					EVENT_STREAM,
-					new EventStream(COLLECTION, TIMESTAMP_PATH, "other", MEMBER_TYPE),
-					new EventStream(COLLECTION, "other", VERSION_OF_PATH, MEMBER_TYPE),
-					new EventStream(COLLECTION, TIMESTAMP_PATH, VERSION_OF_PATH, MEMBER_TYPE),
-					new EventStream(COLLECTION, "other", "other", MEMBER_TYPE),
-					new EventStream(COLLECTION, "other", "other", "other"))
+					new EventStream(COLLECTION, TIMESTAMP_PATH, "other"),
+					new EventStream(COLLECTION, "other", VERSION_OF_PATH),
+					new EventStream(COLLECTION, TIMESTAMP_PATH, VERSION_OF_PATH),
+					new EventStream(COLLECTION, "other", "other"),
+					new EventStream(COLLECTION, "other", "other"))
```

Comment:
There is a duplicate, as there last argument made them different from each other